modules/ding_loan/plugins/content_types/loans.inc

```diff
@@ -51,6 +51,14 @@ function ding_loan_loans_content_type_render($subtype, $conf, $panel_args, $cont
 
       $ids[] = $item->ding_entity_id;
     }
+    ding_entity_load_multiple($ids);
+
+    // Add the pre-load entity to each loan, if the entity exists.
+    foreach ($loans as $id => &$item) {
+      // Accessing the entity ref. will load the entity from the data well. So
+      // this is intentional not assigned to any thing.
+      $item->entity;
```

Comment:
This line actually tries to load an entity, while call on line 54 - `ding_entity_load_multiple($ids);` does exactly the same, which results in same object being tried to be loaded twice. This finding was a part of the lately investigated case about use pages performance.

Reply:
This finding was a part of a more complex investigation for the sake of page load speed, so can't tell without diving into it again.